src/handle.ml

```diff
@@ -161,7 +163,7 @@ let handle_cmd_aux : sig_state -> command -> sig_state * proof_data option =
             (* If the proof is finished, display a warning. *)
             if Proof.finished st then wrn cmd.pos "You should add QED.";
             (* Add a symbol corresponding to the proof, with a warning. *)
-            let s = Sign.add_symbol ss.signature Const x a in
+            let s = Sign.add_symbol ss.signature Const x a [] in
```

Comment:
Implicits are missing here. There may be implicit arguments in theorems too.

Reply:
Done thanks.

Reply:
What is done? Where are the implicits? It shouldn't be [].